libafl_nyx/build.rs

```diff
@@ -0,0 +1,20 @@
+use std::process::Command;
+
+#[cfg(target_os = "linux")]
```

Comment:
Probably this also needs to be `#[cfg(all(target_os = "linux", target_arch = "x86_64"))]`, or is nyx supported on other platforms?

Reply:
not sure, but I guess that the user should know the right OS(`ubuntu`) to run nyx?